### ISSUE TITLE
Application crashes after init

### ISSUE THREAD

**Issue by kuajia** [mapsme/omim#11856]
After the app has been setted up, it crashes with this error 

> 2019-10-09 13:40:37.721345+0200 maps.me dbg[49408:4478009] Task <1F8503AB-7767-4076-ABAB-2CA1851B5B33>.<1> finished with error - code: -1002
> LOG TID(8) DEBUG     40.6141 platform/http_client_apple.mm:182 RunHttpRequest Error:  -1002 : unsupported URL while connecting to 
> ASSERT FAILED
> partners_api/promo_api.cpp:151
> CHECK(!baseUrl.empty()) 
> Assertion failed: (false), function GetCityCatalogueUrl, file /Users/davide/Desktop/XCodeProjects/omim/partners_api/promo_api.cpp, line 151.
> 

This happens after application configuration, giving all permissions and accepting the policy

**Comment by tatiana-yan** [comment]
Hello!
Thank you for report.
See https://github.com/mapsme/omim/pull/11862 PR which should fix the problem.

**Comment by kuajia** [comment]
Thank you Tatiana, I added your code manually, and now I have this issue:

> LOG TID(1) INFO       3411.8 storage/storage.cpp:1367 DownloadNode() Downloading US_California_Santa_Clara_Palo Alto
> 2019-10-09 16:07:36.938347+0200 maps.me dbg[52468:4558054] Task <19518D6C-5B89-471F-B67D-3F97AA6C46E9>.<1> finished with error - code: -1002
> LOG TID(8) DEBUG     3411.83 platform/http_client_apple.mm:182 RunHttpRequest Error:  -1002 : unsupported URL while connecting to 
> LOG TID(8) ERROR     3411.83 platform/servers_list.cpp:29 ParseServerList() Can't parse server list json: 1 '[' or '{' expected near end of file 
> ASSERT FAILED
> base/logging.cpp:108
> CHECK(level < g_LogAbortLevel) 3 3 Abort. Log level is too serious 3
> Assertion failed: (false), function LogMessageDefault, file /Users/davide/Desktop/XCodeProjects/omim/base/logging.cpp, line 108.

**Comment by kuajia** [comment]
Thank for your reply Tatiana. Is there any guide to archive this?

**Comment by kuajia** [comment]
The app crashes on a call to URL METASERVER_URL, which is empty for default

**Comment by tatiana-yan** [comment]
METASERVER_URL is url to get maps servers list

```curl $METASERVER_URL```

should give you json with maps servers list. like

```["http://mapsserver1.com/","http://mapsserver2.com/"]```

If you do not want to use metaserver step you can modify the code and hard-code maps sever to your app.

**Comment by kuajia** [comment]
Is there any guide about modify the code ?

**Comment by tatiana-yan** [comment]
No, there is no specific guide for this. If you want to hardcode server list modify GetServersList from platform. If you want to work with local maps there are lots of ways to do it, it's up to you.
